packages/dashboard/src/components/widgets/tile/tile.tsx

```diff
@@ -75,7 +86,27 @@ const WidgetTile: React.FC<WidgetTileProps> = ({ children, widget, title, remove
             {title}
           </Box>
           <SpaceBetween size='s' direction='horizontal'>
-            {isRemoveable && <DeletableTileAction widget={widget} />}
+            {isRemoveable && <DeletableTileAction handleDelete={handleDelete} />}
+            <ConfirmModal
```

Comment:
we can use the same component in InternalDashboard

Reply:
Used ConfirmDeleteModal component